webpack/stories/docs/ForemanContext.md

```diff
@@ -0,0 +1,50 @@
+# ForemanContext
```

Comment:
can you use the new storybook Text component?
or convert it into an `mdx` file?